core/src/main/scala/pureconfig/BasicReaders.scala

```diff
@@ -109,6 +107,9 @@ trait JavaTimeReaders {
   implicit val periodConfigReader: ConfigReader[Period] =
     ConfigReader.fromNonEmptyString[Period](PeriodUtils.fromString)
 
+  implicit val chronoUnitReader: ConfigReader[ChronoUnit] =
+    configurable.baseJavaEnumReader[ChronoUnit](ConfigFieldMapping(ScreamingSnakeCase, KebabCase))
```

Comment:
Can you add some tests for this new reader?

You may also consider replicating this for the write side so you can do roundtrip tests and keep parity (not required to merge this, though).